aeron-driver/src/main/java/io/aeron/driver/status/SystemCounterDescriptor.java

```diff
@@ -231,7 +231,12 @@ public enum SystemCounterDescriptor
     /**
      * A count of the number of error frames sent by this driver.
      */
-    ERROR_FRAMES_SENT(39, "Error Frames sent");
+    ERROR_FRAMES_SENT(39, "Error Frames sent"),
+
+    /**
+     * A count of the number of IPC rejections encountered by this driver.
+     */
+    IPC_REJECTIONS(40, "IPC rejections");
```

Comment:
I think this should be a more generic counter, i.e. `Images rejected` that should be incremented in both UDP and IPC cases.

Reply:
I like that.  I'll make that change.